Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -184,4 +184,7 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
     private func handle(searchURL: URL?, tabId: String) {
         browserViewController.handle(url: searchURL, tabId: tabId)
     }
+    private func handle(fxaParams: FxALaunchParams) {
```

Comment:
nit, space between func
```suggestion
    }

    private func handle(fxaParams: FxALaunchParams) {
```